understanding_openshift_gitops/about-redhat-openshift-gitops.adoc

```diff
@@ -44,6 +44,13 @@ Argo CD reports any configurations that deviate from their specified state. Thes
 
 // add something about CLI tools?
 
+// Glossary of common terms for OpenShift GitOps
+include::modules/gitops-openshift-go-common-terms.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources 
+* link:https://docs.openshift.com/container-platform/latest/getting_started/openshift-overview.html#getting-started-openshift-common-terms_openshift-overview[What is an OpenShift project?]
+* link:https://argo-cd.readthedocs.io/en/stable/operator-manual/project-specification[Specification of an `AppProject` CRD]
```

Comment:
Just a nitpick, but if we do not include any xrefs, we can have this additional resources section within its module. (Maybe more links are planned to be added in the future?)

Reply:
Also, another thing. Right now there are two additional resources sections after each other. I think that if there is not plan to add more modules as of right now, wouldn't it be better to have just one, probably assembly level, additional resources section?